clj-refactor.el

```diff
@@ -2199,6 +2200,12 @@ the alias in the project."
           (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
             (list short (list long))))))))
 
+(defun cljr--in-sans-ns-keyword-p ()
+  (cljr--sans-ns-keyword?
+   (buffer-substring-no-properties
```

Comment:
Same here, the thing extracted from the buffer is the same as `cider-symbol-at-point` would return.